NebulaWorld/Chat/Commands/HelpCommandHandler.cs

```diff
@@ -43,9 +43,16 @@ private static string GetCommandDetailsOutput(string commandName)
             string aliasList = ChatCommandRegistry.GetCommandAliases(commandName).Join(FullCommandName);
 
             StringBuilder sb = new StringBuilder();
+            foreach(string usage in handler.GetUsage())
+            {
+                sb.Append($"Usage: {FullCommandName(commandName)} {usage}\n");
+            }
+            string usageString = sb.ToString();
```

Comment:
This is a weird way of doing this. Move iterations down and save one string variable allocated.